modules/aws/master/main.tf

```diff
@@ -56,19 +56,14 @@ resource "aws_iam_role_policy" "master_policy" {
   "Version": "2012-10-17",
   "Statement": [
     {
-      "Action": "ec2:Describe*",
+      "Action": "ec2:*",
```

Comment:
Can you drop some notes into the commit message about the new permission you need here?  Does it really need to be `ec2:*`, or is it "`ec2:Describe*` wasn't enough, and I got {error-message}.  I haven't had time to track down the exact new permissions I need, but we can tighten this back down some later".

Reply:
So the machine-api controller and actuator live here. The actuator is the one actually hitting the aws api to create/destroy and operate on ec2 instances and the relevant infra. We can narrow down this later as boundaries are still a bit blurry

Reply:
it would be good to have an issue opened so we do not forget...

Reply:
I'll create one. I wish we had something like kube2iam...

Reply:
I think @sallyom was contemplating doing an openshift version of kube2iam...